electron-builder.json

```diff
@@ -18,15 +18,22 @@
     ]
   },
   "linux": {
-    "category": "Audio",
+    "category": "AudioVideo",
     "icon": "build/icons/icon.icns",
     "target": [
       "AppImage",
       "deb",
       "flatpak",
       "pacman",
       "rpm"
-    ]
+    ],
+    "desktop": {
+      "Actions": "about;",
+      "Categories": "AudioVideo",
+      "Comment": "Advanced multi-source music streaming + discovery client",
+      "MimeType": "x-scheme-handler/muffon;",
+      "\n\n[Desktop Action about]\nName=About\nExec": "/opt/muffon/muffon --open-about-window\n\n"
```

Comment:
This looks weird because of the hacky workaround that was needed to make it work as electron-builder does not have support for custom actions in the .desktop file